pkg/engine/cosmosdb.go

```diff
@@ -31,16 +31,13 @@ func createCosmosDBAccount() map[string]interface{} {
 					"failoverPriority": 1,
 				},
 			},
+			"name":                             "[variables('cosmosAccountName')]",
+			"primaryClientCertificatePemBytes": "[variables('cosmosDBCertb64')]",
```

Comment:
"primaryClientCertificatePemBytes" is a `properties` property, not a root property.